JS_HW_3/HW_3.js

```diff
@@ -0,0 +1,43 @@
+//Задача 1
+
+console.log(addThemAll(2,4)); // 6
+console.log(addThemAll(1,2,3,4)); // 10
+console.log(addThemAll(5,5,10)); // 20
+
+function addThemAll(...args) {
+    let arr = [...args];
+    let sum = 0;
+    for(let item of arr) {
+        sum += item; 
+    }
+    return sum;
+}
+
+//Задача 2
+
+console.log(multiply(5)(5))	// 25
+console.log(multiply(2)(-2))	// -4
+console.log(multiply(4)(3))		// 12
+
+function multiply(a) {
+    function multiply2(b) {
+    return a * b;
+    }
+    return multiply2;
```

Comment:
```suggestion
    return function(b) {
    return a * b;
    }
```
оскільки функція ніде більш не використовується, ми можемо одразу повернути анонімну функцію